README.md

```diff
@@ -629,6 +632,50 @@ var signIn = new OktaSignIn(config);
 // signIn.authClient.options.clientId === '{yourClientId}'
 ```
 
+### before
```

Comment:
Can we add more description about what hooks are for widget.. ? Basically adding following info..

1. Hooks can be added either on run time or via configs as below examples
2. Hooks can be used for ? eg. logging/tracking
3. Hooks blocks processing before/after a login step..